docs/source/migration.ipynb

```diff
@@ -150,9 +150,9 @@
     "And we wish to measure the following quantum observables $O$\n",
     " - the expected value of the 0th bit $O=(1-Z_0)/2$\n",
     " - the expected value of the 1st bit $O=(1-Z_1)/2$\n",
-    " - the expected value of the exclusive or (XOR) between the two qubits $O=(1-ZZ)/2$\n",
+    " - the expected value of the exclusive or (XOR) between the two qubits $O=(1-Z_0Z_1)/2$\n",
     " \n",
-    "**Exercise for the reader:** convince yourself that $(1-ZZ)/2$ is the XOR function"
+    "**Exercise for the reader:** convince yourself that $(1-Z_0Z_1)/2$ is the XOR function"
```

Comment:
I suspect the indexing is supposed to be `Z_1 Z_0`, but will defer to @stylewarning and @AShiTheCoder.

Reply:
they commute so it's ok. If it was a tensor product I'd want it to be Z_1 (x) Z_0. But I'm going to take

```
Z_1 := Z (x) I
Z_0 := I (x) Z
```
and in that lens it's OK.